.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+        "browser": true,
+        "es6": false,
+        "jquery": true,
+        "greasemonkey": true
+    },
+    "extends": [
+        "eslint:recommended"
+    ],
+    "rules": {
+        "eqeqeq": "error",
+        "indent": ["warn", 2],
+        "brace-style": ["error","1tbs", { "allowSingleLine": true }],
+        "keyword-spacing": "error",
+        "spaced-comment": "error",
+        "quotes": ["error", "single", { "avoidEscape": true }],
+        "no-trailing-spaces": "error",
+        "max-len": ["warn", { "code": 160 }],
+        "semi": "error",
```

Comment:
It would be more convenient to have the same rules order as we have is style guide: https://github.com/IITC-CE/ingress-intel-total-conversion/wiki/Code-Style

Reply:
it's a configuration file - noone will every 'read' these files. There is a loosly connetion to the wikipage not a dependency.